src/SFML/Graphics/RenderTextureImplFBO.cpp

```diff
@@ -82,52 +100,214 @@ bool RenderTextureImplFBO::isAvailable()
 
 
 ////////////////////////////////////////////////////////////
-bool RenderTextureImplFBO::create(unsigned int width, unsigned int height, unsigned int textureId, bool depthBuffer)
+bool RenderTextureImplFBO::create(unsigned int width, unsigned int height, unsigned int textureId, const ContextSettings& settings)
 {
+    // Store the dimensions
+    m_width = width;
+    m_height = height;
+
+    // Disable creation of depth/stencil surfaces in the context
+    ContextSettings contextSettings(settings);
+    contextSettings.depthBits = 0;
+    contextSettings.stencilBits = 0;
+    contextSettings.antialiasingLevel = 0;
+    contextSettings.sRgbCapable = false;
+
     // Create the context
-    m_context = new Context;
+    m_context = new Context(contextSettings, 1, 1);
 
-    // Create the framebuffer object
-    GLuint frameBuffer = 0;
-    glCheck(GLEXT_glGenFramebuffers(1, &frameBuffer));
-    m_frameBuffer = static_cast<unsigned int>(frameBuffer);
-    if (!m_frameBuffer)
-    {
-        err() << "Impossible to create render texture (failed to create the frame buffer object)" << std::endl;
+    if (settings.antialiasingLevel && !(GLEXT_framebuffer_multisample && GLEXT_framebuffer_blit))
         return false;
+
+    if (settings.stencilBits && !GLEXT_packed_depth_stencil)
+        return false;
+
+#ifndef SFML_OPENGL_ES
+
+    // Check if the requested anti-aliasing level is supported
+    if (settings.antialiasingLevel)
+    {
+        GLint samples = 0;
+        glGetIntegerv(GLEXT_GL_MAX_SAMPLES, &samples);
+
+        if(settings.antialiasingLevel > samples)
```

Comment:
I would suggest something like this here: `err() << "Impossible to create render texture (unsupported anti-aliasing level, maximum supported samples: " << samples << " )" << std::endl;`